LiveBlip/Components/Pages/Welcome.razor

```diff
@@ -1,18 +1,88 @@
-﻿@page "/welcome"
+﻿
+@page "/welcome"
+@inject NavigationManager NavManager
 
 <PageTitle>Welcome</PageTitle>
 
-<div class="welcome-card">
-	<div class="page-heading">
-		<h1>Welcome to</h1>
-		<img src="images/logo.png" class="logo"
-	</div>
 
-	<div class="input-imei">
-		<input type="text" class="form-control-imei" placeholder="Please input tracker IMEI" />
-	</div>
+<MudPaper Class="mx-auto my-16 pa-6 mud-theme-dark border-solid border-2 mud-border-primary pa-4 " Height="230px" Width="50%" Elevation="22" Outlined="true" >
```

Comment:
Please remove the border, the red one, it doesn't really follow the rest of the theme